tests/upgrade/postgres_upgrade_run.sh

```diff
@@ -0,0 +1,422 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091
+
+set -euo pipefail
+
+# Tests upgrade to Postgres.
+
+TEST_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../.. && pwd)"
+
+EARLIER_TAG="4.7.2"
+EARLIER_SHA="ef2060ed332c7b8513cb9eb52b9745df9a8285cc"
+CURRENT_TAG="$(make --quiet --no-print-directory tag)"
+PREVIOUS_RELEASES=("4.1.3" "4.2.5" "4.3.8" "4.4.8" "4.5.9" "4.6.5" "4.7.2")
```

Comment:
This looks like a total copy of the existing test.  Do we want to just replace the existing one with this one OR should we trim this one down if we want to keep them separate?  Meaning I wouldn't think we would want to go through all the previous releases if the purpose here is to simply test PG13 <-> PG 15.  In fact would it make sense for this to be temporary as it is?

Reply:
Good point, I think it would be sufficient to have 4.7.2 and 4.8 versions.